awsiot/greengrass_discovery.py

```diff
@@ -34,16 +35,23 @@ def __init__(
             bootstrap: ClientBootstrap,
             socket_options: SocketOptions,
             tls_context: ClientTlsContext,
-            region: str):
+            region: str,
+            gg_server_name = None):
```

Comment:
trivial: add type hint for new arg, since there are type hints for all the other args

`gg_server_name: str = None):`